apps/prairielearn/src/lib/externalGrader.ts

```diff
@@ -216,6 +219,13 @@ export async function processGradingResult(content: any): Promise<void> {
       }
     }
 
+    const grading_job = await selectOptionalGradingJob(content.gradingId);
```

Comment:
Nit, trailing `ById`? I know we're still establishing conventions here, so maybe we don't want that in new functions.

Reply:
Added in 3fa4c4533a7c6af1ac1d6884c644ddbb98de39a0

It would be good to document our conventions at some point, but sticking with `ById()` for now seems reasonable.